bridge/contracts/InvalidTxConsumptionAccusation.sol

```diff
@@ -63,6 +63,8 @@ contract InvalidTxConsumptionAccusation is
         if (pClaims.bClaims.txCount == 0) {
             revert AccusationsErrors.NoTransactionInAccusedProposal();
         }
+
+        // todo: check if the height is at least 2 epochs behind
```

Comment:
is this still necessary?

Reply:
Yes, and it shouldn't be addressed on this PR :/